core/HomeButton.java

```diff
@@ -0,0 +1,46 @@
+package edu.asu.c3simulator.widgets;
+
+import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.files.FileHandle;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.ui.Image;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+
+import edu.asu.c3simulator.screens.MainHub;
+//import edu.asu.c3simulator.screens.MainHub;
+/**
+ * This class creates a Home button actor, places it in the bottom left of the screen. Changes the screen to MainHub when clicked.
+ * 
+* @author Reigel, Justin 
+* 
+*/
+public class HomeButton extends Table{
```

Comment:
This looks a lot like a button that is given a specific position and listener. It doesn't seem to add any functionality or properties that are not contained in Button.

I propose that we replace this class with a method in WidgetFactory